dotnet/build-packages.sh

```diff
@@ -1,8 +1,8 @@
 
-RELEASE_VERSION="2.1.0"
-DEVELOPMENT_VERSION="0.1.58-prerelease"
+RELEASE_VERSION="2.2.0"
```

Comment:
What's the reasoning between minor/patch here? As it's a hotfix, should we make the AAS bump a hotfix too?

Reply:
Did that because irt to AAS, I believe this is not just a hotfix. 
As we said, it will be hard to keep those in sync, I don't think it was an issue

Reply:
Yeah sure, no problem with them not being in sync, just seems like a hotfix to me if it only includes a Tracer hotfix 😄 Happy either way, just wondered